contributing/code/core_team.rst

```diff
@@ -13,6 +13,17 @@ This document states the rules that govern the Symfony core team. These rules
 are effective upon publication of this document and all Symfony Core members
 must adhere to said rules and protocol.
 
+Role of a Core Member
+---------------------
+
+In addition to being a regular contributor, core members are expected to:
+
+ * Review, approve, and merge pull requests;
+
+ * Help enforce, improve, and implement Symfony :doc:`processes and policies </contributing/index>`;
+
+ * Participate in the Symfony Core Team discussions (on Slack and GitHub).
```

Comment:
@xabbuh @javiereguiluz Shouldn't there be no blank line between each item? Not sure if there's a guideline on this point in the doc, but it seems that most of the time, there's not blank line between list items

Reply:
I don't think we have a rule for it, but it indeed seems that we often don't have blank lines between items.

Reply:
You are right. We've removed those lines and other extra lines in some lists of the same doc article here: https://github.com/symfony/symfony-docs/commit/6301562c49f02e2a15b253dd007aa55ad0c336cc

Thanks!